utility/importer/brave_importer.h

```diff
@@ -0,0 +1,58 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_UTILITY_IMPORTER_BRAVE_IMPORTER_H_
+#define BRAVE_UTILITY_IMPORTER_BRAVE_IMPORTER_H_
+
+#include <stdint.h>
+
+#include <map>
+#include <vector>
+
+#include "base/files/file_path.h"
+#include "base/macros.h"
+#include "base/nix/xdg_util.h"
+#include "base/values.h"
+#include "build/build_config.h"
+#include "chrome/common/importer/imported_bookmark_entry.h"
+#include "chrome/utility/importer/importer.h"
+
+class BraveImporter : public Importer {
```

Comment:
We should inherit ChromeImporter and make `Import*` virtual functions so that we don't have to duplicate codes for `ImportPasswords` and `ImportCookies`